code/modules/items/gimmick/figurine.dm

```diff
@@ -35,16 +35,26 @@
 
 			var/potential_donator_ckey = usr?.mind?.ckey
 			var/donator_figtype = null
+			var/list/figure_patreon_online_nouser = figure_patreon_online
```

Comment:
`=` isn't a copy operation, so any modifications here *will* be reflected in the global list. Not sure what "nouser" means but I'm guessing this isn't what you want